tensorflow/core/kernels/gpu_utils.cc

```diff
@@ -282,6 +282,34 @@ Status BestCudnnConvAlgorithm(absl::Span<const AutotuneResult> results,
   return Status::OK();
 }
 
+Status BestCudnnConvExecutionPlan(
```

Comment:
Can you merge this function and `BestCudnnConvAlgorithm`, now that AlgorithmConfig is merged?

Reply:
Done.